io.github.lime3ds.Lime3DS.json

```diff
@@ -0,0 +1,170 @@
+                        "project-id": 334920,
+                        "url-template": "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "rapidjson",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DRAPIDJSON_ENABLE_INSTRUMENTATION_OPT=OFF",
+                "-DRAPIDJSON_BUILD_DOC=OFF",
+                "-DRAPIDJSON_BUILD_EXAMPLES=OFF",
+                "-DRAPIDJSON_BUILD_TESTS=OFF",
+                "-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz",
+                    "sha256": "bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "stable-only": true,
+                        "project-id": 7422,
+                        "url-template": "https://github.com/Tencent/rapidjson/archive/v$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "lime",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_C_COMPILER=clang",
+                "-DCMAKE_CXX_COMPILER=clang++",
+                "-DCMAKE_LINKER=lld",
+                "-DENABLE_QT_TRANSLATION=ON",
+                "-DCITRA_ENABLE_COMPATIBILITY_REPORTING=ON",
+                "-DUSE_DISCORD_PRESENCE=ON",
+                "-DENABLE_COMPATIBILITY_LIST_DOWNLOAD=OFF",
+                "-DUSE_SYSTEM_SDL2=ON"
+            ],
+            "cleanup": [
+                "/share/man",
+                "/share/pixmaps"
+            ],
+            "post-install": [
+                "install -Dm755 ../lime3ds-launcher.sh /app/bin/lime3ds-launcher",
+                "install -Dm644 ../io.github.lime3ds.Lime3DS.metainfo.xml /app/share/appdata/io.github.lime3ds.Lime3DS.metainfo.xml",
+                "desktop-file-install --dir=/app/share/applications ../dist/lime-qt.desktop",
+                "desktop-file-edit --set-key StartupWMClass --set-value lime-qt --set-key Exec --set-value lime3ds-launcher /app/share/applications/lime-qt.desktop",
+                "install -Dm644 ../io.github.lime3ds.Lime3DS.svg /app/share/icons/hicolor/scalable/apps/lime.svg",
+                "install -Dm644 ../dist/icon.png /app/share/icons/hicolor/512x512/apps/lime.png",
+                "mv /app/share/mime/packages/lime.xml /app/share/mime/packages/io.github.lime3ds.Lime3DS.xml",
+                "sed 's/lime/io.github.lime3ds.Lime3DS/g' -i /app/share/mime/packages/io.github.lime3ds.Lime3DS.xml"
```

Comment:
you should use `rename-mime-icons` (take an array of icon name) in the top level of the manifest. It's new in flatpak-builder 1.4

sed is bad.